src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.except.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final String EXPECTED_EXCEPTION =
+            InvalidDataException.class.getSimpleName();
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static InvalidDataException invalidDataException;
+
+    @BeforeAll
+    static void beforeAll() {
+        RegistrationServiceImplTest registrationServiceImplTest = new RegistrationServiceImplTest();
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_positiveTest_ok() {
+        User user = new User();
+        user.setAge(19);
+        user.setLogin("User");
+        user.setPassword("123456");
+        storageDao.add(user);
+    }
```

Comment:
we should create new users and add them to database before each test

Reply:
I need them only in one test case